tests/lava/lib/peripherals/realsense/test_realsense_input.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import unittest
+import subprocess
+import pyrealsense2 as rs
+import numpy as np
+
+from lava.magma.core.run_configs import Loihi2SimCfg
+from lava.magma.core.run_conditions import RunSteps
+
+from lava.lib.peripherals.realsense.realsense import (
+    DirectRealsenseInput,
+    DirectRealsenseInputPM,
+)
+
+
+class TestPyRealSenseCameraModel(unittest.TestCase):
+    def test_base_functionality_file(self):
+        num_steps = 10
+        height = 360
+        width = 640
+        rgbdcamera = DirectRealsenseInput(
```

Comment:
What does the process do without a filename?

Reply:
depth alignment needs a color frame. If a depth camera doesn't have a color sensor, the alignment can't work properly.
added this comments in code